cmd/get/launch_plan.go

```diff
@@ -27,10 +28,15 @@ Retrieves launch plan by name within project and domain.
 
  flytectl get launchplan -p flytesnacks -d development core.basic.lp.go_greet
 
-Retrieves launchplan by filters.
+Retrieves all the launchplan with filters.
 ::
 
- Not yet implemented
+ bin/flytectl get launchplan -p flytesnacks -d development core.basic.lp.go_greet --field-selector="launchplan.name=core.basic.lp.go_greet"
```

Comment:
Similarly here args and filed-selector both could be confusing.

Also add example with composite selection with how to access fields from launchplan object for filtering and sorting.